frontend/src/hocs/withHostCommunication/withHostCommunication.tsx

```diff
@@ -87,7 +88,7 @@ function withHostCommunication(
         if (
           !origin ||
           message.stCommVersion !== HOST_COMM_VERSION ||
-          !CLOUD_COMM_WHITELIST.find(el => isValidURL(el, origin))
+          !allowedOrigins.find(el => isValidURL(el, origin))
```

Comment:
Can you add a comment to this entire `if` block describing briefly why it's important? (Like, what does it even mean if a message is coming from a different origin and why should the program care.)